.github/workflows/integration-tests.yaml

```diff
@@ -11,13 +11,28 @@ defaults:
     shell: bash
 
 jobs:
+  prepare:
```

Comment:
can we call this job "fetch-latest-keptn-versions" or alike?

Reply:
Sure just changed it.